.github/workflows/CICD.yml

```diff
@@ -0,0 +1,21 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
```

Comment:
I think we might need a test job as well. But at this stage in the project, this could be enough.

Reply:
But what can we test in test step? npm run server? Or maybe “jobs: test: -run: npm test” ?

Reply:
I am not sure about that.
What do you think about the test? And why did you choose to only add build?

Reply:
Because I didn't know what we should test in CI when we were talking about the task, I remembered we only mentioned the CI is to make sure the code is working when pushing to a branch, in my understanding, when the code can npm start(here is npm build) and create a "dist" it will work well. So I didn't put a test step at here